sql/qaqc/qaqc_de.sql

```diff
@@ -0,0 +1,61 @@
+DROP TABLE IF EXISTS DE_qaqc;
```

Comment:
Could we put "app" in the same to differentiate the two tables by their usage/destination? Could rename the other to put "HPD_review" or something in the same